torch_xla/experimental/distributed_checkpoint/manager.py

```diff
@@ -1,9 +1,17 @@
+import os
+import torch.distributed as dist
 import torch.distributed.checkpoint as dist_cp
+import torch_xla.runtime as xr
 import torch_xla.experimental.distributed_checkpoint as xc
 
-from typing import List, Optional
+from fsspec.core import url_to_fs
+from os.path import basename
+from typing import List, Optional, Union
 from torch.distributed.checkpoint.metadata import STATE_DICT_TYPE
 
+# TODO(jonbolin): Import path will change
+from torch.distributed.checkpoint._fsspec_filesystem import FsspecReader, FsspecWriter
```

Comment:
The import path will change when the API becomes public in the upstream. @alanwaketan @yeounoh do you have any thoughts on how to handle this?

Reply:
It's okay. The upstream test will break our CI in the upstream, and then we can have a companion change to fix it.